packages/build/src/error/type.js

```diff
@@ -158,6 +159,13 @@ const TYPES = {
     rawStack: true,
     severity: 'error',
   },
+
+  // Errors related with the telemetry output
+  telemetry: {
+    showInBuildLog: false,
+    title: 'Telemetry error',
```

Comment:
Since this is a system error, we might want to re-use the following properties:
  - `stackType: 'stack'`: prints the stack trace
  - `rawStack: true`: does not clean-up the stack trace. Cleaning up the stack trace is meant for user errors, so users have an easier to stack trace to tack actions upon. For system errors, we want as much information in the stack trace as possible.
  - `showErrorProps: true`: print any static properties present on the error. `got()` adds some of those, which might be useful here.

Reply:
Addressed via https://github.com/netlify/build/pull/2439/commits/02c7a4ecefa23773fad402505c15edcb3a1e6b6d